controllers/workspace/devworkspace_controller.go

```diff
@@ -667,9 +667,8 @@ func (r *DevWorkspaceReconciler) getWorkspaceId(ctx context.Context, workspace *
 	}
 }
 
-func (r *DevWorkspaceReconciler) SetupWithManager(mgr ctrl.Manager) error {
-	setupHttpClients()
-
+func (r *DevWorkspaceReconciler) SetupWithManager(mgr ctrl.Manager, k8s client.Client) error {
+	setupHttpClients(k8s)
```

Comment:
We can use `mgr.GetClient()` instead of passing another parameter.

Reply:
done